rdf-delta-server-local/src/main/java/org/seaborne/delta/server/local/patchstores/PatchLogBase.java

```diff
@@ -175,12 +174,8 @@ public Version append(RDFPatch patch) {
         });
     }
 
-    protected void patchLogLock(Runnable action) {
-        logIndex.runWithLock(action);
-    }
-
     protected <X> X patchLogLockRtn(Supplier<X> action) {
-        return logIndex.runWithLockRtn(action);
+        return logIndex.runWithLock(action);
```

Comment:
> Because of the way signatures are resolved, it has to be a different name.

That's not the case because a `Supplier<X>` is easily distinguishable from a `Runnable` as evidenced by the facts that everything compiled and that the tests passed despite this change. However, I can change it back if you really want me to seeing as it's your project.